wildfly/extensions/src/main/java/org/keycloak/provider/wildfly/ConfigProviderFactoryPropertyOverrideSupport.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2019 Red Hat, Inc. and/or its affiliates
+ * and other contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.keycloak.provider.wildfly;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+import javax.servlet.ServletContext;
+import org.jboss.resteasy.spi.ResteasyProviderFactory;
+
+public interface ConfigProviderFactoryPropertyOverrideSupport {
+
+    static final String SERVER_CONTEXT_CONFIG_PROPERTY_OVERRIDES = "keycloak.server.context.config.property-overrides";
+
+    default public Map<String, String> getPropertyOverrides() {
+
+        ServletContext context = ResteasyProviderFactory.getContextData(ServletContext.class);
+        Map<String, String> propertyOverridesMap = new HashMap<>();
+        String propertyOverrides = context.getInitParameter(SERVER_CONTEXT_CONFIG_PROPERTY_OVERRIDES);
+
+        try {
+            if (context.getInitParameter(SERVER_CONTEXT_CONFIG_PROPERTY_OVERRIDES) != null) {
+                JsonNode jsonObj = new ObjectMapper().readTree(propertyOverrides);
```

Comment:
Ideally, we should always use org.keycloak.util.JsonSerialization for parsing Json using Jackson. An object mapper instance is already there.